README.md

```diff
@@ -252,6 +256,16 @@ config = TransferConfig(multipart_threshold=5*1024*1024*1024)
 self.boto3_client.upload_file(put_filename, bucket_name, key, Config=config)
 ```
 
+### Example of MinIO mc client
+
+The `mc cp` command has a `--disable-multipart` option for file uploads.  Here is an example of an upload with a `myminio` alias:
+
+```bash
+mc cp --disable-multipart put_file myminio/bucket_name/put_filename
+```
+
+> Note: MinIO client uses aliases to group URL and keys. Refer to the `mc alias` command for information on setting, listing, and removing aliases.
```

Comment:
Why a quote box instead of bold or italicized text?
Is it common for quote boxes to be used in README files to highlight text?

Reply:
i don't find it too awful.  but maybe 'Note:' could just be bolded.   don't feel strongly either way.

Reply:
I'm not against it, but it is the first time we've considered using quote boxes in a README AFAIK.

My personal preference would be to make it bold or italic so that it matches other parts of the document. I also understand this is just 0.1.0 so it's not a big deal.

I'll leave the final decision up to @JustinKyleJames. Feel free to leave it as is if you like.

Reply:
I changed it to italics.